BadHabbit/BadHabbit/UserStories/SingUpScreen/SignUpViewController.swift

```diff
@@ -36,6 +36,17 @@ final class SignUpViewController: BaseViewController, SignUpViewProtocol {
 
         setupUI()
     }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+       navigationController?.setNavigationBarHidden(true, animated: animated)
+
+    }
+
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+       navigationController?.setNavigationBarHidden(false, animated: animated)
```

Comment:
Fix the alignment in viewWillAppear(_:) and viewWillDisappear(_:) functions

Reply:
resolved